config/lib/render.js

```diff
@@ -0,0 +1,4 @@
+/**
+  Template rendering function
+*/
+module.exports = require('consolidate').swig;
```

Comment:
Nice move!

Actually Swig is quite a bit outdated and I'd like to replace it with Nunjucks or similar, but I'll wait for this before doing anything about it: https://github.com/meanjs/mean/issues/1286

Just sayin, in the scope of this PR Swig is still okay.